src/components/styles/Theme/ThemeProvider.tsx

```diff
@@ -161,7 +161,23 @@ interface NavbarTheme {
     minHeight: string;
     bgColor: string;
   };
+  logoContainer?: {
+    desktop: {
```

Comment:
Not 100% on the names here - happy for input?

`minMediaDesktop` might be better to be accurate here